centreon/www/front_src/src/Authentication/Local/LocalAuthentication.cypress.spec.tsx

```diff
@@ -0,0 +1,603 @@
+import { renderHook } from '@testing-library/react-hooks/dom';
+import { useAtomValue } from 'jotai';
+import { BrowserRouter as Router } from 'react-router-dom';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+import { replace } from 'ramda';
+
+import { userAtom } from '@centreon/ui-context';
+import { buildListingEndpoint, Method, TestQueryProvider } from '@centreon/ui';
+
+import {
+  authenticationProvidersEndpoint,
+  contactsEndpoint
+} from '../api/endpoints';
+import { Provider } from '../models';
+
+import {
+  labelReset,
+  labelDefinePasswordPasswordSecurityPolicy,
+  labelDoYouWantToResetTheForm,
+  labelNumberOfAttemptsBeforeUserIsBlocked,
+  labelPasswordBlockingPolicy,
+  labelPasswordCasePolicy,
+  labelPasswordExpirationPolicy,
+  labelMinimumPasswordLength,
+  labelResetTheForm,
+  labelSave,
+  labelPasswordMustContainLowerCase,
+  labelPasswordMustContainUpperCase,
+  labelPasswordMustContainNumbers,
+  labelPasswordMustContainSpecialCharacters,
+  labelStrong,
+  labelGood,
+  labelWeak,
+  labelPasswordExpiresAfter,
+  labelMonth,
+  labelDays,
+  labelMinimumTimeBetweenPasswordChanges,
+  labelHour,
+  labelExcludedUsers,
+  labelChooseADurationBetween7DaysAnd12Months,
+  labelChooseADurationBetween1HourAnd1Week,
+  labelLast3PasswordsCanBeReused,
+  labelTimeThatMustPassBeforeNewConnection,
+  labelMinutes,
+  labelChooseAValueBetween1and10,
+  labelThisWillNotBeUsedBecauseNumberOfAttemptsIsNotDefined,
+  labelBlockingDurationMustBeLessThanOrEqualTo7Days,
+  labelDay
+} from './translatedLabels';
+import {
+  defaultPasswordSecurityPolicyAPI,
+  retrievedPasswordSecurityPolicyAPI,
+  defaultPasswordSecurityPolicyWithNullValues,
+  securityPolicyWithInvalidDelayBeforeNewPassword,
+  securityPolicyWithInvalidPasswordExpiration,
+  securityPolicyWithInvalidBlockingDuration
+} from './defaults';
+import { PasswordSecurityPolicyToAPI } from './models';
+
+import LocalAuthentication from '.';
+
+dayjs.extend(duration);
+
+const LocalAuthenticationTestWithJotai = (): JSX.Element => (
+  <TestQueryProvider>
+    <LocalAuthentication />
+  </TestQueryProvider>
+);
+
+const setComponentBeforeEach = (): void => {
+  const defaultPasswordSecurityPolicyURL = authenticationProvidersEndpoint(
+    Provider.Local
+  );
+
+  cy.interceptAPIRequest<PasswordSecurityPolicyToAPI>({
+    alias: 'getDefaultPasswordSecurityPolicyFromAPI',
+    method: Method.GET,
+    path: replace('./', '**', defaultPasswordSecurityPolicyURL),
+    response: defaultPasswordSecurityPolicyAPI
+  });
+
+  cy.mount({
+    Component: (
+      <Router>
+        <div style={{ backgroundColor: '#fff' }}>
+          <LocalAuthenticationTestWithJotai />
+        </div>
+      </Router>
+    )
+  });
+
+  cy.viewport(1200, 1000);
+};
+
+before(() => {
+  document.getElementsByTagName('body')[0].style = 'margin:0px';
```

Comment:
We must move it into the `mount` custom command

Reply:
This involves modifying all the tests that already call on this command, should I integrate it into this PR ?